model-optimizer/mo/utils/ir_reader/extenders/deconvolution_extender.py

```diff
@@ -25,6 +25,9 @@ def extend(op: Node):
 
 
 def common_backpropdata_extender(op: Node):
+    for attr in ['strides', 'output_padding', 'pads_begin', 'pads_end', 'dilations']:
+        Extender.attr_to_list(op, attr)
```

Comment:
I don't understand why do you call this routine if these attribute values are already of list type. Please check the specification.

Reply:
If these attributes restored ir there are some cases then they can have `int` type (for 1D case) and we need to check it and convert to list as Yegor does. We have a practice to resolve such cases in operations extenders. You can find an example of it here: `model-optimizer/mo/utils/ir_reader/extenders/conv_extender.py`